lib/rules/no-property-in-node.js

```diff
@@ -55,13 +56,34 @@ module.exports = {
       requiresTypeChecking: true,
       url: 'https://github.com/eslint-community/eslint-plugin-eslint-plugin/tree/HEAD/docs/rules/no-property-in-node.md',
     },
-    schema: [],
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          additionalNodeTypeFiles: {
+            description:
+              'Any additional regular expressions to consider source files defining AST Node types.',
+            default: '^(enforce|require|disallow)',
```

Comment:
the `default` was copied from https://github.com/eslint-community/eslint-plugin-eslint-plugin/blob/04e397b6d3c02084a68f5339f182fb5aecaaca5b/lib/rules/require-meta-docs-description.js#L32? 😄

Reply:
Haha you got me